website/pages/en/index.js

```diff
@@ -74,115 +71,211 @@ HomeSplash.propTypes = {
+    <img src={image} style={{ width: "100px", height: "100px" }} />
+    <div style={{ flexGrow: 1, paddingLeft: "8px" }}>
+      <h3 style={{ marginBottom: "-16px" }}>{name}</h3>
+      <MarkdownBlock>{content}</MarkdownBlock>
+    </div>
+  </div>
+);
+
+Editor.propTypes = {
+  content: React.PropTypes.string,
+  image: React.PropTypes.string.isRequired,
+  name: React.PropTypes.string.isRequired
+};
+
+const EditorSupportSection = () => (
+  <div className="editorSupportSection productShowcaseSection lightBackground paddingTop paddingBottom">
+    <Container>
+      <h2>Editor Support</h2>
+      <div
+        style={{
+          display: "flex",
+          flexFlow: "row wrap",
+          justifyContent: "space-around"
+        }}
+      >
+        {siteConfig.editors.map(editor => (
+          <Editor key={editor.name} {...editor} />
+        ))}
+      </div>
+    </Container>
+
+    <div style={{ float: "right" }}>
+      <span>Got more? </span>
+      <a
+        href={`${siteConfig.githubUrl}/edit/master/website/data/editors.yml`}
+        className="button"
+      >
+        Send a PR
+      </a>
+    </div>
+  </div>
+);
+
+const bash = string => `~~~bash\n${string}\n~~~`;
+
+const json = object => `~~~json\n${JSON.stringify(object, null, 2)}\n~~~`;
+
+class GetStartedSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      npmClient: "yarn"
+    };
+  }
+
+  render() {
+    return (
+      <div className="getStartedSection productShowcaseSection paddingTop paddingBottom">
+        <Container>
+          <h2>Get Started</h2>
+          <div>
+            <ol>
+              <li>
+                Add prettier to your project:
+                <MarkdownBlock>
+                  {bash(`yarn add prettier --dev --exact`)}
+                </MarkdownBlock>
+              </li>
+              <li>
+                Verify by running against a file:
+                <MarkdownBlock>
+                  {bash(`yarn prettier --write src/index.js`)}
+                </MarkdownBlock>
+              </li>
+              <li>
+                Run prettier when commiting files:
+                <MarkdownBlock>
+                  {bash(`yarn add pretty-quick husky --dev`)}
+                </MarkdownBlock>
+                Then edit <code>package.json</code>:
+                <MarkdownBlock>
+                  {json({
+                    scripts: {
+                      precommit: "pretty-quick --staged"
+                    }
+                  })}
+                </MarkdownBlock>
+              </li>
+            </ol>
+          </div>
+        </Container>
+      </div>
+    );
+  }
+}
+
+const UsersSection = ({ language }) => {
+  const showcase = siteConfig.users
+    .filter(user => {
+      return user.pinned;
+    })
+    .map((user, i) => {
+      return (
+        <a key={i} href={user.infoLink}>
+          <img
+            src={user.image}
```

Comment:
Some of these images don’t look very good on a dark background; we should either switch to a light background or get inverted versions of the images.